Modelica/Electrical/Polyphase/Sensors/AronSensor.mo

```diff
@@ -75,9 +75,10 @@ Contains two
 to measure total active power in a three-phase system.
 </p>
 <p>
-This device works only for sinusoidal voltages and currents in three-phase systems without neutral.
+This device works only in three-phase systems without neutral.
```

Comment:
This change is perfectly OK as sinusoidal voltages and currents are not an assumption. Solely the neutral must not be connected.

Reply:
Just to be sure I understand well: the latter comment of yours means that you agree with my correction without me to have to search for a reference?

The Aron connection for me is a straightforward consequence of the Tellegen's theorem, 
which just requires that a circuit can be formulated as the composition of two subcircuits connected to each other through three wires. No need to consider sinusoidal shapes.

In case you are curious or deem it useful I can add details of this.

Reply:
Yes, I fully agree with fixing line 78 and your arguments as the original documentation is wrong.